lms/djangoapps/certificates/apps.py

```diff
@@ -20,3 +22,6 @@ def ready(self):
         # Can't import models at module level in AppConfigs, and models get
         # included from the signal handlers
         from . import signals  # pylint: disable=unused-variable
```

Comment:
Unused import

Reply:
This is not an unused import. We are registering our signals here.